server/app/views/components/ButtonStyles.java

```diff
@@ -97,7 +96,9 @@ public final class ButtonStyles {
   public static final String LINK_STYLE =
       StyleUtils.joinStyles(
           StyleUtils.removeStyles(ApplicantStyles.LINK, "text-sm"),
-          "bg-transparent",
           "border-none",
           "hover:bg-opacity-0");
+
+  public static final String LINK_STYLE_WITH_TRANSPARENCY =
```

Comment:
Why: Same issues with `bg-transparency`. Unfortunately there some places need it, some don't. Creating this new variable to be the one with transparency and updating the couple spots that need it. The existing one without `bg-transparency` stays on all other existing buttons.